precompiles/gov/GovI.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.17;
+
+/// @dev The GovI contract's address.
+address constant GOV_PRECOMPILE_ADDRESS = 0x0000000000000000000000000000000000000805;
+
+/// @dev Define all the available gov methods.
+string constant MSG_VOTE = "/cosmos.gov.v1.MsgVote";
+
+/// @dev The GovI contract's instance.
+GovI constant GOV_CONTRACT = GovI(
+    GOV_PRECOMPILE_ADDRESS
+);
+
+/**
+ * @dev VoteOption enumerates the valid vote options for a given governance proposal.
+ */
+enum VoteOption {
+    // Unspecified defines a no-op vote option.
+    Unspecified,
+    // Yes defines a yes vote option.
+    Yes,
+    // Abstain defines an abstain vote option.
+    Abstain,
+    // No defines a no vote option.
+    No,
+    // NoWithWeto defines a no with veto vote option.
+    NoWithWeto
+}
+
+/// @author Luke
+/// @title Gov Precompile Contract
+/// @dev The interface through which solidity contracts will interact with Gov
+/// @custom:address 0x0000000000000000000000000000000000000805
+interface GovI {
```

Comment:
Can you please rename this `IGovernance` instead and also make the file with the same name